jinja2/includes/login.html

```diff
@@ -1,11 +1,29 @@
 <div class="login">
     {% if user.is_authenticated() %}
-        <a class="user-url" href="{{ user.get_absolute_url() }}">{{ user_display(user) }}</a>
-        <form action="{{ url('account_logout') }}" method="post">
-            {% csrf_token %}
-            <input name="next" type="hidden" value="{{ next_url }}">
-            <button class="logout" type="submit">{{ _('Sign out') }}</button>
-        </form>
+        {% if waffle.flag('redesign') %}
+
+            <a class="user-url" href="{{ user.get_absolute_url() }}"><img src="{{ gravatar_url(user.email, size=50) }}" alt="{{ user_display(user) }}" width="50" height="50" class="login-photo avatar" /><span class="login-name">{{ user_display(user) }}</span> <i aria-hidden="true" class="icon-caret-down"></i></a>
```

Comment:
Nit: `height` on the `img` tag is optional unless we want to enforce the aspect ratio, which might very be the case.